test_runner/fixtures/zenith_fixtures.py

```diff
@@ -277,24 +263,48 @@ def generate_tenant_token(self, tenant_id):
         return token
 
 
+@zenfixture
+def worker_seq_no(worker_id: str):
+    if worker_id == 'master':
+        return 0
+    assert worker_id.startswith('gw')
+    return int(worker_id[2:])
+
+
+@zenfixture
+def worker_base_port(worker_seq_no: int):
+    # so we divide ports in ranges of 100 ports
```

Comment:
Hmm, BASE_PORT is 55000, and the max port number is 65535, so this will break at about 105 workers. That seems enough, but maybe add an explicit check for that, so that if you get a nice error message if you try to run with `pytest -n 200`

Reply:
Also would be good to add a check that if you run out of the 100 ports reserved for a single test, you get a nice error message about that too.